docker/elasticsearch-setup/create-indices.sh

```diff
@@ -97,9 +97,22 @@ function create_datahub_usage_event_aws_elasticsearch() {
 }
 
 if [[ $DATAHUB_ANALYTICS_ENABLED == true ]]; then
+  echo -e "\n datahub_analytics_enabled: $DATAHUB_ANALYTICS_ENABLED"
   if [[ $USE_AWS_ELASTICSEARCH == false ]]; then
     create_datahub_usage_event_datastream || exit 1
   else
     create_datahub_usage_event_aws_elasticsearch || exit 1
   fi
+else
+  echo -e "\ndatahub_analytics_enabled: $DATAHUB_ANALYTICS_ENABLED"
+  DATAHUB_USAGE_EVENT_INDEX_RESPONSE_CODE=$(curl -o /dev/null -s -w "%{http_code}" --header "$ELASTICSEARCH_AUTH_HEADER" "$ELASTICSEARCH_PROTOCOL://$ELASTICSEARCH_HOST:$ELASTICSEARCH_PORT/cat/indices/${PREFIX}datahub_usage_event")
+  if [ $DATAHUB_USAGE_EVENT_INDEX_RESPONSE_CODE -eq 404 ]
+  then
+    echo -e "\ncreating ${PREFIX}datahub_usage_event"
+    curl -XPUT --header "$ELASTICSEARCH_AUTH_HEADER" "$ELASTICSEARCH_PROTOCOL://$ELASTICSEARCH_HOST:$ELASTICSEARCH_PORT/${PREFIX}datahub_usage_event"
```

Comment:
I think this needs the same `$ELASTICSEARCH_INSECURE_MODE` prefix to the url as the rest of the curl calls.

Reply:
add `${ELASTICSEARCH_INSECURE}` to curl calls (https://github.com/datahub-project/datahub/pull/5974/commits/d1c231a9992addcfdedcddaf66761998e5f608ae)